index.js

```diff
@@ -9,6 +9,7 @@ const fs = require("fs");
 // config
 const defaultConfig = require("./default-config.json");
 const config = require("./local/config.json");
+const forbidden = require("./local/forbidden-commands.json");
```

Comment:
wydaje mi sie że zamiast nowywego pliku lepiej będzie by w komendzie była zmienna czy można jej używać w PM'ach

Reply:
Also consider, że mamy już inny styl require'owania w index.js..

Reply:
to luz ja juz załatwie mergowanie

Reply:
to bardzo łatwo można wpisać w index

Reply:
chyba nawet samemu to możesz dodać